changelogs/fragments/1460-sysrc-new-module.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- sysrc - new module plugin for managing rc files on FreeBSD (https://github.com/ansible-collections/community.general/pull/1460).
```

Comment:
Could you please remove this? New module/plugin PRs must not have a changelog fragment (except for test/filter plugins).

Reply:
Removed